utils/m0mount

```diff
@@ -403,7 +403,8 @@ echo "Device:" > disks.conf
 devs=\`ls $DISKS_PATTERN | grep -v part\`
 
 for dev in \$devs; do
-	partitions=\$(partprobe -ds \$dev 2>/dev/null)
+	partitions=\$(partprobe -ds \$dev 2>/dev/null |
+                      sed -e 's/.*partitions//' -e '/^$/d')
```

Comment:
Because we are generating a shell script by "cat" command.
I hope we don't need to escape the special chars (^, $) here.

Reply:
No, we don't. It works.

Reply:
OK